10/umbraco-deploy/deploy-settings.md

```diff
@@ -152,9 +156,12 @@ All of these times are configured using [standard timespan format strings](https
 
 Even with appropriate settings of the above timeouts, Deploy's backoffice transfer operations can hit a hard limit imposed by the hosting environment. For Azure, this is around 4 minutes. This will typically only be reached if deploying a considerable amount of items in one go. For example, a media folder with thousands of items can reach this limit.
 
-An error message of `500 - The request timed out. The web server failed to respond within the specified time.` will be reported.
+An error message of "The remote API has returned a response indicating a platform timeout" will be reported.
 
-If encountering this issue, the `SourceDeployBatchSize` setting can be applied with an integer value (for example 1000). This will cause Deploy to transfer items in batches, up to a maximum size. This will allow each individual batch to complete within the time available.
+If encountering this issue, there are two batch settings that can be applied with integer values (for example 500). This will cause Deploy to transfer items in batches, up to a maximum size. This will allow each individual batch to complete within the time available.
+
+- `SourceDeployBatchSize` - applies a batch setting for the transfer of multiple selected items to an upstream environment (such as a media folder with many images).
+- `PackageBatchSize` - applies a batch setting to the processing of a Deploy "package", which contains all the items selected for a Deploy operation, plus all the determined dependencies and relations.
```

Comment:
And for this setting:
> A batch setting is also available for the processing of a package. A package is an ordered structure containing all the items selected for the operation, plus all the determined dependencies and relations.
> 
> When set, if the number of items determined for the package exceeds the batch size, the processing will be chunked into batches. Although this will result in a single deployment, the batches are processed twice: the second reprocessing operation ensures consistency of the deployed content (e.g. when a dependency didn't exist yet during the initial processing of a batch).
> 
> This will take effect for both transfers to and (partial) restores from upstream environments, regardless of the number of items selected.